backend/src/main/java/gov/cdc/usds/simplereport/config/AuthorizationConfiguration.java

```diff
@@ -30,13 +30,22 @@
           + AUTHORIZER_BEAN
           + ".userHasPermission("
           + "T(gov.cdc.usds.simplereport.config.authorization.UserPermission).";
+
+  private static final String SPEL_IS_SITE_ADMIN =
+      "@" + AUTHORIZER_BEAN + ".userHasSiteAdminRole()";
+
+  private static final String SPEL_IS_NOT_SELF = "@" + AUTHORIZER_BEAN + ".userIsNotSelf(#userId)";
+
+  private static final String SPEL_IS_NOT_IN_SAME_ORG =
+      "@" + AUTHORIZER_BEAN + ".userIsInSameOrg(#userId)";
```

Comment:
Shouldn't this be named `SPEL_IS_IN_SAME_ORG`? This tripped me up on [line 67](https://github.com/CDCgov/prime-simplereport/pull/1037/files#diff-b5255ef1f759cdfb6352cd1e0a6ca5d1df0e8230844a97464a997e4aa09d5fb8R67)

Reply:
oopsie